app/src/main/res/layout-land/drag_drop_interaction_item.xml

```diff
@@ -32,7 +32,7 @@
     app:explorationViewMarginApplicable="@{viewModel.hasConversationView &amp;&amp; !viewModel.splitView}"
     app:explorationViewMarginBottom="@{@dimen/space_0dp}"
     app:explorationViewMarginEnd="@{@dimen/drag_drop_item_exploration_view_margin_end}"
-    app:explorationViewMarginStart="@{@dimen/drag_drop_interaction_item_exploration_view_margin_start}"
+    app:explorationViewMarginStart="@{@dimen/drag_drop_interaction_item_exploration_view_padding_start}"
```

Comment:
Please revert this change as this PR is not focused on `DragAndDrop`, also this PR does not exists anymore in develop and its name should be `drag_drop_interaction_item_exploration_view_margin_start ` instead of `drag_drop_interaction_item_exploration_view_padding_start `

Reply:
Done.